src/board/Board.jsx

```diff
@@ -0,0 +1,19 @@
+import Row from "./Row";
+import "./Board.css";
+import PropTypes from "prop-types";
+
+function Board({ data }) {
+  return (
+    <div className="board">
+      {data.map((row, index) => (
+        <Row key={index} row={row} data-testid="row" />
+      ))}
+    </div>
+  );
+}
+
+Board.propTypes = {
+  data: PropTypes.arrayOf(PropTypes.arrayOf(PropTypes.number)).isRequired,
+};
```

Comment:
An alternative to this type of import is to import the arrayOf and number from the library so it is much more readable. This may be a matter of preference for your team though.

Reply:
this makes sense. Thank you. It definitely makes things more readable.